cpp/include/raft/neighbors/ivf_pq.cuh

```diff
@@ -296,6 +465,66 @@ void extend(raft::device_resources const& handle,
   detail::extend(handle, idx, new_vectors, new_indices, n_rows);
 }
 
+/**
+ * @brief Extend one list of the index in-place, by the list label, skipping the classification and
```

Comment:
Can we get brief usage examples for these as well that a user could copy/paste into their code?